erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.py

```diff
@@ -204,6 +211,9 @@ def get_invoice_entries(self):
 			accounting_dimensions=self.accounting_dimension_filter_conditions,
 		)
 
+		cr_dr_notes = [x.voucher_no for x in self.return_invoices]
```

Comment:
HR tests for Payment Reco with Expense Claim failing after this PR: 
https://github.com/frappe/hrms/actions/runs/5230489656/jobs/9468777474?pr=586 

`self.return_invoices` is only set for parties: customer & supplier in `get_dr_or_cr_notes` so references to `self.return_invoices` fail.

Reply:
Checking

Reply:
addressed: https://github.com/frappe/erpnext/pull/35659